.eslintrc.json

```diff
@@ -0,0 +1,32 @@
+{
+  "env": { "browser": true, "es6": true, "jest": true },
+  "settings": {
+    "import/resolver": { "webpack": { "config": "./webpack.common.js" } }
+  },
+  "extends": [
+    "airbnb",
+    "prettier",
+    "plugin:jsx-a11y/recommended",
+    "plugin:cypress/recommended"
+  ],
+  "plugins": ["react", "jsx-a11y"],
+  "globals": { "Atomics": "readonly", "SharedArrayBuffer": "readonly" },
+  "parserOptions": {
+    "ecmaFeatures": { "jsx": true },
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "rules": {
+    "one-var": 0,
```

Comment:
@Ramadhan0 all the rules set to 0 are turned off. Please, read the configuration docs and be sure of what the rules do: https://eslint.org/docs/user-guide/configuring

Also, sync with @nkpremices since he implemented Eslint for the backend repo